sycl/include/sycl/ext/intel/experimental/grf_size_properties.hpp

```diff
@@ -0,0 +1,91 @@
+//==- grf_size_properties.hpp - GRF size kernel properties for Intel GPUs -==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===--------------------------------------------------------------------===//
+
+#pragma once
+
+#include <sycl/ext/oneapi/properties/property.hpp>
+#include <sycl/ext/oneapi/properties/property_value.hpp>
+
+#define SYCL_EXT_INTEL_GRF_SIZE 1
+
+namespace sycl {
+__SYCL_INLINE_VER_NAMESPACE(_V1) {
+namespace ext::intel::experimental {
+struct grf_size_key {
+  template <unsigned int Size>
```

Comment:
I don't think I can add it right here because it's an alias template, but I was able to add it somewhere else and I just pushed that change, good point, thanks.